chef/cookbooks/network/recipes/default.rb

```diff
@@ -92,6 +92,13 @@
   subscribes :run, resources(cookbook_file: "modprobe-bridge.conf"), :delayed
 end
 
+# Increase inotify max user instances
+# https://bugs.launchpad.net/charms/+bug/1593041
+execute "increasing inotify max user instances" do
+  command "echo 1024 > /proc/sys/fs/inotify/max_user_instances"
+  action :nothing
```

Comment:
This would not survive any reboots right? Plus rigth now its not doing anything :)

Reply:
That's a good point @Itxaka. @abel-navarro we need to notify this resource somewhere or just run it.